tests/testMultiTypeTable.m

```diff
@@ -0,0 +1,326 @@
+classdef testMultiTypeTable < matlab.unittest.TestCase
+%%
+% testMultiTypeTable Class based unit tests for the multiTypeTable Class
+% used within the Project Class in RAT.
+%
+% In this class, we test:
+% multiTypeTable, addRow, setValue, appendNewRow, removeRow,
+% displayTypesTable, findRowIndex
+%
+% We use an example multi-type table from the backgrounds class for the
+% example calculation "DPPC_standard_layers.m"
+%
+% Paul Sharp 30/01/23
+%
+%% Declare properties and parameters
+
+    properties (TestParameter)
+        % Cell arrays for initialising a multi type table
+        inputCell = {{},...
+                     {'Row 0'},...
+                     {'Row 0', 'constant'},...
+                     [{'Row 0', 'constant'}, repmat({''}, 1, 1)],...
```

Comment:
All your parameters use `constant` only even though there are 3 other options. Would be good to add variation

Reply:
I've put some in for the "addRow" tests.